_includes/devdoc/guide_dash_features.md

```diff
@@ -730,6 +730,24 @@ voted to delete them.
 Sentinel manages superblock creation, voting, and submission to dashd for
 network propagation.
 
+Beginning ~3 days (1662) blocks prior to a superblock, Sentinel selects one
```

Comment:
> 3 days (1662) blocks

Maybe move the word "blocks" inside the parens?

Reply:
Oops, yep that's where it was supposed to be :-)